dev-docs/guides/transaction_imports.md

```diff
@@ -58,35 +58,35 @@ Incoming fee reimbursements (refunding people for the credit card fees Stripe de
+That’s how invoices and donations can be imported once they leave Stripe. But how do we tell them to leave Stripe? To do this, we create Stripe payouts from our balance when an invoice or donation is paid. This logic can be found in `InvoicePayout` and `DonationPayout`.
 
 #### Reimbursements
 
-We use a clearinghouse organisation for reimbursements. This means that the ACHs / checks etc. we send to reimburse people are standard ACHs / checks. Just like the one any organisation would send! See `Reimbursement::PayoutHoldingService::Nightly` for how this is done.
+We use a clearinghouse organisation for reimbursements. This means that the ACHs, checks, etc. we send to reimburse people are standard ACHs / checks, just like the ones any organisation would send! See `Reimbursement::PayoutHoldingService::Nightly` for how this is done.
 
 The transactions that are imported by HCB short code are the `ExpensePayout` and the `PayoutHolding`. These are internal book transfers that move money from the organisation that is reimbursing someone to the clearinghouse organisation.
 
 An `ExpensePayout` is a book transfer from “FS Main” to “FS Operating”. It comes in as a negative `CanonicalTransaction`. A `Reimbursement::Report` can have multiple of these `ExpensePayout`s, one for every `Reimbursement::Expense` that was approved. 
 
-An `PayoutHolding` is a book transfer from “FS Operating” to “FS Main”. It comes in as a positive `CanonicalTransaction`. A `Reimbursement::Report` can only have one `PayoutHolding`. It’s `amount_cents` will be the sum of the `Reimbursement::Report`’s `ExpensePayout`s’ `amount_cents`.
+A `PayoutHolding` is a book transfer from “FS Operating” to “FS Main”. It comes in as a positive `CanonicalTransaction`. A `Reimbursement::Report` can only have one `PayoutHolding`. Its `amount_cents` will be the sum of the `Reimbursement::Report`’s `ExpensePayout`s’ `amount_cents`.
 
 ### Subledgers
 
-Events can have subledgers. For example, a card grant. When mapping these transactions to an event, we map them to the event as usual but add an additional `subledger_id` to the `CanonicalEventMapping`. Not all transaction importing mechanisms support this at the moment. However, they are supported by HCB short code mapping (“guessed” based on the HCB code’s other transactions), disbursements (based on the `Disbursement`’s `source_subledger_id` and `destination_subledger_id`) and Stripe card transactions (based on the `StripeCard`’s `subledger_id`).
+Events can have subledgers. For example, a card grant. When mapping these transactions to an event, we map them to the event as usual but add an additional `subledger_id` to the `CanonicalEventMapping`. Not all transaction importing mechanisms support this at the moment. However, they are supported by HCB short code mapping, disbursements (based on the `Disbursement`’s `source_subledger_id` and `destination_subledger_id`), and Stripe card transactions (based on the `StripeCard`’s `subledger_id`).
```

Comment:
This "guessing" still happens if there's only one event / it isn't a disbursement